apiserver/admin_test.go

```diff
@@ -486,20 +486,6 @@ func (s *loginSuite) infoForNewUser(c *gc.C, info *api.Info) *api.Info {
 	return &newInfo
 }
 
-func (s *loginSuite) TestNonModelUserLoginFails(c *gc.C) {
```

Comment:
This one I couldn't see how the old functionality mapped to the new and came to the conclusion that it was a redundant test. I believe the old test also has a misleading title compared to the functionality.

What it actually does is add a user to the model, then remove their user access from the controller. This means the user no longer has login access because it is no longer in the controller user collection.

Now I look I've realized that my assumption that this was tested elsewhere was wrong. I will put up a PR to add the test back in.